vcd/resource_vcd_snat_test.go

```diff
@@ -125,11 +135,27 @@ func testAccCheckVcdSNATDestroy(s *terraform.State) error {
 }
 
 const testAccCheckVcdSnat_basic = `
+resource "vcd_network_routed" "{{.OrgVdcNetworkName}}" {
+  name         = "{{.OrgVdcNetworkName}}"
+  org          = "{{.Org}}"
+  vdc          = "{{.Vdc}}"
+  edge_gateway = "{{.EdgeGateway}}"
+  gateway      = "{{.Gateway}}"
+
+  static_ip_pool {
+    start_address = "{{.StartIpAddress}}"
+    end_address   = "{{.EndIpAddress}}"
+  }
+}
+
+
 resource "vcd_snat" "{{.SnatName}}" {
   org          = "{{.Org}}"
   vdc          = "{{.Vdc}}"
   edge_gateway = "{{.EdgeGateway}}"
+  network_name = "{{.OrgVdcNetworkName}}"
```

Comment:
Do we still have a test without `network_name` after that? We may easily break the older functionality until 3.0 if we don't.

Reply:
Addressed